### DISCUSSION TITLE
Firefly-iii 6.0.21 CSS not loading properly

### DISCUSSION THREAD

**Discussion by pinalgirkar** [firefly-iii/firefly-iii#7916] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I had successfully installed firefly-iii 6.0.21 on {{Alma Linux 8.8 with Apache/2.4.54 , PHP(8.2) and MySQL(5.7)}}, however after complete installation when accessing the admin-panel the CSS doesn't load properly.

Followed this documentation to install firefly-6.0.21 https://docs.firefly-iii.org/firefly-iii/installation/self-hosted

Screenshot has been attached for the reference
![firefly](https://github.com/firefly-iii/firefly-iii/assets/106646072/51fb3d61-b6a9-469c-9c96-71e33a88229c)

### Debug information

--

### Expected behaviour

It should load the admin panel as shown in the screenshot
![image](https://github.com/firefly-iii/firefly-iii/assets/106646072/2b0ab066-4b5f-417e-8358-aeb9083d3b1a)

### Steps to reproduce

1. Install the firefly-iii 6.0.21 using the following step as mentioned in the documentation (https://docs.firefly-iii.org/firefly-iii/installation/self-hosted)
2. Then visit the admin link and register the user and access the admin panel

**Comment by JC5** [comment]
If you visit the source of the page, what is the URL of the "base href"?

**Reply by pinalgirkar** [reply]
When visiting the source page{{URL_TO_INSTALLATION}}/public/new-user , while visiting the links then it redirects to the root 
domain

**Debug_informatiom**
Debug information generated at 2023-09-01 13:42:57 Europe/Amsterdam for Firefly III version **6.0.21**.

| Scope | Version |
| --- | --- |
| Firefly III | 6.0.21 |
| Firefly III API | 2.0.6 |
| DB version | 20 (exp. 20) |
| Docker | No |
| PHP | `8.2.9` |
| Host | `Linux` |


| System info | Value |
| --- | --- |
| System TZ | Europe/Amsterdam |
| Browser TZ | Asia/Calcutta |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, daily |
| Display errors | Off |
| Error reporting | ALL errors |
| Max upload | 20971520 (20 MB) |
| Interface | fpm-fcgi |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, sqlite |
| Current driver | mysql |
| Trusted proxies (.env) | `` |

| User info | Value |
| --- | --- |
| Login provider | eloquent  |
| User guard | web  |
| Headers | N/A, N/A  |
| Stateful domains |  |
| Session start |  |
| Session end |  |
| Session first |  |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36 |

**Reply by JC5** [reply]
Press "view page source" and tell me what it says in `<base href="">`

**Comment by pbanj** [comment]
having the same issue. some of the pages are fine, but some dont load css at all
page source shows
`<base href="http://mydomain/">`

i tried https://docs.firefly-iii.org/firefly-iii/faq/installation/#i-get-page-load-errors-because-the-protocols-dont-match and it didnt make a difference
